api4/bot_test.go

```diff
@@ -592,13 +605,29 @@ func TestGetBots(t *testing.T) {
 
 		bots, resp := th.Client.GetBotsIncludeDeleted(2, 2, "")
 		CheckOKStatus(t, resp)
-		require.Equal(t, []*model.Bot{deletedBot2}, bots)
+		require.Equal(t, []*model.Bot{deletedBot2, orphanedBot}, bots)
 
 		botList := model.BotList(bots)
 		bots, resp = th.Client.GetBotsIncludeDeleted(2, 2, botList.Etag())
 		CheckEtag(t, bots, resp)
 	})
 
+	t.Run("get bots, page=0, perPage=10, only orphaned", func(t *testing.T) {
```

Comment:
I confess that I don't entirely understand the potential parallelism of go subtests, can you confirm that this change of role permissions is not racy?